docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
 Run of Show
 ===========
 
-We are still working out the run of show for the virtual event.
+This Run of Show provides more context about the event and answers some common questions you may have. 
+Please let us know if there is any information missing that would be useful for you.
+
+Sponsorship schedule
+--------------------
+
+* **SUNDAY**: The conference online platform is open. You are welcome to hang look at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also welcome to lead a Writing Day project if your documentation is open source.
+
+* **MONDAY**: The conference platform opens at 8am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the main conference talks, when sponsors tables will need to be staffed. The conference will run until around 5pm.
+
+* **TUESDAY**: The Job Fair will be Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon. 
+
+See the :doc:`full schedule </conf/{{ shortcode }}/{{ year }}/schedule>` for exact timing details.
+
+Sponsorship platform
+--------------------
+
+We will be using `Hopin <https://hopin.to/>`_ as our online conference platform. It allows a number of unique spaces for attendees during the conference, and we hope it will allow a good amount of interaction between attendees and sponsors.
+
+**We will open the online platform 1 week prior to the conference starting**. At that point, you will be able to login and look around the platform, but it won't become fully active until the Sunday of the conference.
+
+Hopin has a set of `sponsor information <https://support.hopin.to/en/articles/3574463-vendor-instructions>`_ that explains a bit more about the sponsorship setup on the platform.
+
+Sponsorship spaces
+------------------
+
+A quick overview of the spaces in the venue that are important:
+
+* The *main stage* is where the talks happen. This is also where Lightning talks will be given.
+* The *session area* is where the Unconference & Job Fair will happen. Sponsors will be assigned booths at the scheduled time for their job fair, and booths will be available for the duration of the conference.
```

Comment:
Thought: can we influence ordering of sessions in hopin's display? Showing job fair sessions at the top of the list? Or at least visible enough?

Reply:
"Sponsors will be assigned booths at the scheduled time for their job fair, and booths will be available for the duration of the conference." -> this seems contradictory. The job fair session is available for the scheduled time, the expo booth is available for the whole conference, right? Perhaps the part after the comma should move to the next bullet.